src/forking-store.mjs

```diff
@@ -1,47 +1,29 @@
-"use strict";
+import rdflib, { Fetcher, UpdateManager, namedNode, Statement } from "rdflib";
```

Comment:
The real problem is that rdflib doesn't have a default export, so that's why `_rdflib.default.something` doesn't work. We need to use `import * as rdflib` here or use named exports instead.